migrations/23_add_proxy_fields_to_contracts.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE contracts
+ADD is_proxy BOOLEAN DEFAULT false;
```

Comment:
on second thought, this field is already implied by having a non-NULL value in `proxy_for`, so I don't think we need it